### PR TITLE
Fix/autocomplete

### PR DESCRIPTION
**Pull request by axe312ger**

### CODE REVIEW

File: themes/docs/src/components/live-editor/autocompletion.ts [hunk 1]
```diff
@@ -36,6 +36,18 @@ interface Range {
...
+function getPropertySnippet({ name, type }, index = 1) {
+  if (type === 'bool') {
+    return name
+  }
+
+  if (type === 'object') {
+    return `${name}={{\${${index}:}}}`
+  }
+
+  return `${name}="\${${index}:}"`
```
Comment (zcei) [inline themes/docs/src/components/live-editor/autocompletion.ts:48]: I would make this string specific and default to `{\${1:}}` (`name={<cusor>}`) as this is the most generic form to pass a value to a JSX prop.

Comment (axe312ger) [reply themes/docs/src/components/live-editor/autocompletion.ts:48]: @zcei but bool attributes not going to have `=`

Comment (axe312ger) [reply themes/docs/src/components/live-editor/autocompletion.ts:48]:
```
function getPropertySnippet({ name, type }, index = 1) {
  let cursor = `="\${${index}:}"`
  
  if (type === 'bool') {
    cursor = ``
  }

  if (type === 'object') {
    return `={{\${${index}:}}}`
  }

  return `${name}${cursor}`
}

```

🤔 🙅

Comment (zcei) [reply themes/docs/src/components/live-editor/autocompletion.ts:48]: Yes, but whaddaboud numbers? Or arrays?

Comment (axe312ger) [reply themes/docs/src/components/live-editor/autocompletion.ts:48]: Not used within our components yet 🙃 

There are many more prop type we don't properly support. Like `.anyOf()` and others.

Comment (zcei) [reply themes/docs/src/components/live-editor/autocompletion.ts:48]: `anyOf` is just a propsType validation. But if you say none of the above is justed, then ✅

**Review by zcei — APPROVED** [review]

**Pull request merged**